addons/stock/models/stock_move_line.py

```diff
@@ -777,6 +777,8 @@ def get_aggregated_properties(move_line=False, move=False):
 
         # Does the same for empty move line to retrieve the ordered qty. for partially done moves
         # (as they are splitted when the transfer is done and empty moves don't have move lines).
+        if 'package' in kwargs:
```

Comment:
Could you use the form `if kwargs.get('package')` in case we call the method with a falsy value ? 
Also, I think this behavior of excluding 'empty move lines' could be useful in other situations. So I would rather name the parameter `strict`